### PR TITLE
Update the description for rw_worker_nodes

### LINKED ISSUE #1543: Document: refactor(rw_catalog): make rw_worker_nodes list all nodes and resource
This issue tracks the documentation update needed for the merged PR #13487.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/13487
Source PR Merged At: 2023-11-20T07:33:50Z

### PR DESCRIPTION
**Pull request by WanYixian**
## Info

- **Description**

Update the description for `rw_worker_nodes` in the table: 

1. Add nodes: meta node, compute node, frontend node, compactor node.

2. Add info: `rw_version`, `total_memory_bytes`, `total_cpu_cores` and `started_at`. In the updated doc, their names are versions, memory bytes, CPU cores, and start timestamps.

- **Related code PR**

   https://github.com/risingwavelabs/risingwave/pull/13487

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1543

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/sql/system-catalogs/rw_catalog.md [hunk 1]
```diff
@@ -142,4 +142,4 @@ SELECT name, initialized_at, created_at FROM rw_sources;
...
  rw_users              | Contains information about users that are available in the database, including their unique IDs, names, and boolean flags indicating whether they are a superuser, whether they can create databases, whether they can create other users, and whether can log in.|
  rw_views              | Contains information about views that are available in the database, including their unique IDs, names, schema IDs, owner IDs, definitions, and more.|
- rw_worker_nodes       | Contains information about worker nodes that are available in the database, including their unique IDs, hostnames, ports, types, states, parallelism levels, and boolean flags indicating whether they are used for streaming, serving, or are unschedulable.|
+ rw_worker_nodes       | Contains information about worker nodes that are available in the database (meta nodes, compactor nodes, compute nodes, and frontend nodes), including their unique IDs, hostnames, ports, types, states, parallelism levels, boolean flags indicating whether they are used for streaming, serving, or are unschedulable, versions, memory bytes, CPU cores, and start timestamps.|
```
Comment (hengm3467) [inline docs/sql/system-catalogs/rw_catalog.md]: The description is getting a bit too long to be contained in one sentence. Can we rearrange the elements introduced and/or break it into several smaller pieces. Thanks.

Comment (WanYixian) [reply docs/sql/system-catalogs/rw_catalog.md]: Thanks for the comment. Break it into two sentences now, pls check.

File: docs/sql/system-catalogs/rw_catalog.md [hunk 2]
```diff
@@ -142,4 +142,4 @@ SELECT name, initialized_at, created_at FROM rw_sources;
...
  rw_users              | Contains information about users that are available in the database, including their unique IDs, names, and boolean flags indicating whether they are a superuser, whether they can create databases, whether they can create other users, and whether can log in.|
  rw_views              | Contains information about views that are available in the database, including their unique IDs, names, schema IDs, owner IDs, definitions, and more.|
- rw_worker_nodes       | Contains information about worker nodes that are available in the database, including their unique IDs, hostnames, ports, types, states, parallelism levels, and boolean flags indicating whether they are used for streaming, serving, or are unschedulable.|
+ rw_worker_nodes       | Contains information about worker nodes available in the database, such as meta nodes, compactor nodes, compute nodes, and frontend nodes. The detailed information includes their unique IDs, hostnames, ports, types, states, parallelism levels, boolean flags (indicating whether they are used for streaming, serving, or are unschedulable), versions, memory bytes, CPU cores, and start timestamps.|
```
Comment (zwang28) [inline docs/sql/system-catalogs/rw_catalog.md]: maybe:
versions -> kernel versions
memory bytes, CPU cores -> memory, CPU
start timestamps -> uptime

Comment (WanYixian) [reply docs/sql/system-catalogs/rw_catalog.md]: Revised now, pls check, thanks

**Review by hengm3467 — APPROVED** [review]
Thanks! The updated description looks good to me.

**Review by zwang28 — APPROVED** [review]
LGTM

**Pull request merged**